examples/with-edgedb/README.md

```diff
@@ -0,0 +1,134 @@
+
+```bash
+$  npm install
+# or
+$  yarn
+```
+
+### Install the CLI
+
+First install the EdgeDB CLI if you haven't already.
+
+```bash
+# macOS/Linux
+$ curl --proto '=https' --tlsv1.2 -sSf https://sh.edgedb.com | sh
+
+# windows
+$ iwr https://ps1.edgedb.com -useb | iex
+```
+
+### Initialize the EdgeDB project
+
+Initialize the project with the following CLI command:
+
+```bash
+$ edgedb project init
+```
+
+After you follow the prompts, this command will spin up a local EdgeDB instance and apply all the migrations inside `dbschema/migrations`. Now that the project is initialized, all EdgeDB clients initialized inside the project directory will connect to this instance automatically—no need for environment variables or hard-coded configuration. ([Read more about projects here.](https://www.edgedb.com/docs/guides/projects))
+
+### Generate the query builder
+
+This project uses the EdgeQL query builder for TypeScript. This tool can express any EdgeQL query in a code-first way and infers a static return type. Generate it with the following command:
+
+```bash
+$ npx edgeql-js
+```
+
+The query builder consists of several files that are generated into the `dbschema/edgeql-js` directory. Import it like so:
+
+```ts
+import e from './dbschema/edgeql-js'
+```
+
+### Seed the database
+
+```bash
+$ npx ts-node seed.ts
+```
+
+### Start the app
+
+```bash
+$ yarn dev
+```
+
+The application should now be running on http://localhost:3000.
+
+## Notes
+
+### API structure
+
+- `POST /api/post`: Create a new post
+  - Body: `{title: string; content: string; authorName: string}`
+- `PATCH /api/post/:id`: Update a post by `id`
+  - Body: `{title?: string; content?: string;}`
+- `PUT /api/publish/:id`: Publish a post by `id`
+- `DELETE /api/post/:id`: Delete a post by `id`
+
+## Evolving the app
+
+Evolving the application typically requires three steps:
+
+1. Update the schema in `dbschema/default.esdl`
+1. Generate a new migration with `edgedb migration create`
+1. Apply the migration with `edgedb migrate`
+1. Regenerate the query builder with `npx edgeql-js`
+
+## Deployment
+
+To deploy this application, follow the EdgeDB deployment guides for your preferred cloud provider:
+
+- [AWS](https://www.edgedb.com/docs/guides/deployment/aws_aurora_ecs)
+- [Google Cloud](https://www.edgedb.com/docs/guides/deployment/gcp)
+- [Azure](https://www.edgedb.com/docs/guides/deployment/azure_flexibleserver)
+- [DigitalOcean](https://www.edgedb.com/docs/guides/deployment/digitalocean)
+- [Fly.io](https://www.edgedb.com/docs/guides/deployment/fly_io)
+- [Docker](https://www.edgedb.com/docs/guides/deployment/docker) (cloud-agnostic)
+
+After setting up your cloud-hosted EdgeDB instance:
+
+1. Deploy this app to Vercel with the button below.
+2. Find your instance's DSN (AKA connection string). The exact instructions for this depend on which cloud you are deploying to.
+3. Set the value of `EDGEDB_DSN` environment variable via the Vercel dashboard.
```

Comment:
Seems like this variable should be set before the first deploy, it can be set during the project creation flow by changing the URL like below

Reply:
Updated